Python/traceback.c

```diff
@@ -926,4 +1008,3 @@ _Py_DumpTracebackThreads(int fd, PyInterpreterState *interp,
 
     return NULL;
 }
-
```

Comment:
Aah applying the github suggestion wiped this blank line.

Reply:
Probably my bad, it was a rough merge with all the stuff changed. I think you should have commit privs on colnotab, so feel free to fix these irrelevant stuff in commit